saleor/graphql/schema.graphql

```diff
@@ -4083,6 +4083,18 @@ enum PostalCodeRuleInclusionTypeEnum {
   EXCLUDE
 }
 
+type PreorderData {
+  isPreorder: Boolean!
+  globalThreshold: Int
+  globalSoldUnits: Int
+  endDate: DateTime
+}
+
+type PreorderThreshold {
+  quantity: Int
```

Comment:
Is this field nullable?

Reply:
Yes, it is - by definition. User can set threshold to None - it means there is no threshold per channel and only the global threshold should be used (if it is set).